test/e2e_node/resource_metrics_test.go

```diff
@@ -84,20 +84,30 @@ var _ = framework.KubeDescribe("ResourceMetricsAPI", func() {
 					fmt.Sprintf("%s::%s::%s", f.Namespace.Name, pod1, "busybox-container"): boundedSample(0, 100),
 				}),
 
-				"container_memory_working_set_bytes": gstruct.MatchAllElements(containerID, gstruct.Elements{
+				"container_memory_working_set_bytes": gstruct.MatchElements(containerID, gstruct.IgnoreExtras, gstruct.Elements{
 					fmt.Sprintf("%s::%s::%s", f.Namespace.Name, pod0, "busybox-container"): boundedSample(10*e2evolume.Kb, 80*e2evolume.Mb),
 					fmt.Sprintf("%s::%s::%s", f.Namespace.Name, pod1, "busybox-container"): boundedSample(10*e2evolume.Kb, 80*e2evolume.Mb),
 				}),
 			})
 			ginkgo.By("Giving pods a minute to start up and produce metrics")
-			gomega.Eventually(getV1alpha1ResourceMetrics, 1*time.Minute, 15*time.Second).Should(matchV1alpha1Expectations)
+			gomega.Eventually(fetcher, 1*time.Minute, 15*time.Second).Should(matchResourceMetrics)
 			ginkgo.By("Ensuring the metrics match the expectations a few more times")
-			gomega.Consistently(getV1alpha1ResourceMetrics, 1*time.Minute, 15*time.Second).Should(matchV1alpha1Expectations)
+			gomega.Consistently(fetcher, 1*time.Minute, 15*time.Second).Should(matchResourceMetrics)
+		}
+
+		ginkgo.It("should report resource usage through the alpha resource metrics api", func() {
+			resourceFetcher := getV1alpha1ResourceMetrics
+			fetchAndTest(f, resourceFetcher)
+		})
+		ginkgo.It("should report resource usage through the stable resource metrics api", func() {
+			resourceFetcher := getResourceMetrics
+			fetchAndTest(f, resourceFetcher)
```

Comment:
We can't say the `/metrics/resource/v1alpha1` is the alpha resource metrics  API and `/metrics/resource` is not the stable resource metrics API, this will make confusion. They are both the resource metrics API in different endpoints.